paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/cdc/CdcMultiplexRecordChannelComputer.java

```diff
@@ -51,9 +57,13 @@ public void setup(int numChannels) {
 
     @Override
     public int channel(CdcMultiplexRecord multiplexRecord) {
+        int recordChannel =
+                computeChannelComputer(multiplexRecord) != null
```

Comment:
reuse `computeChannelComputer(multiplexRecord)` to a local field

Reply:
done